src/serverdata.rs

```diff
@@ -0,0 +1,209 @@
+// This file is generated by rust-protobuf 3.3.0. Do not edit
+// .proto file is parsed by protoc --rust-out=...
```

Comment:
Should prob ignore this?

Reply:
I dont think itll compile in github for our cd checks

Reply:
ah ok nvm then.